.github/CODEOWNERS

```diff
@@ -20,11 +20,11 @@ tests/snapshots/                    @DataDog/apm-python
 riotfile.py                         @DataDog/apm-python
 
 # Core
-tests/internal                      @DataDog/apm-core-python
+tests/internal                      @DataDog/python-guild
 tests/lib-injection                 @DataDog/apm-core-python
-tests/opentelemetry                 @DataDog/apm-core-python
-tests/opentracer                    @DataDog/apm-core-python
-tests/runtime                       @DataDog/apm-core-python
+tests/opentelemetry                 @DataDog/python-guild
+tests/opentracer                    @DataDog/apm-python
```

Comment:
All of the opentelemetry and opentracer files should be assigned to the API team.

cc @mabdinur who might be able to help us figure out which github team to assign here.

Reply:
@brettlangdon @mabdinur what is the "API team" referring to here?

Reply:
@emmettbutler SDK APIs, the new tracing team (the "not IDM" team)

Reply:
@brettlangdon I can't find any team with a name like that in the Github teams list https://github.com/orgs/DataDog/teams?query=api

Reply:
@emmettbutler yeah, they need to create the GitHub team.